.eslintrc.js

```diff
@@ -142,7 +142,9 @@ module.exports = {
         'src/test/workspaceSymbols/main.unit.test.ts',
         'src/test/workspaceSymbols/generator.unit.test.ts',
         'src/test/datascience/extensionapi/exampleextension/ms-ai-tools-test/src/serverPicker.ts',
-        'vscode.d.ts'
+        'vscode.d.ts',
+        'vscode.proposed.*.d.ts',
+        '.eslintrc.js'
```

Comment:
Since `.eslintrc.js` is edited by us, i'd not exclude this file from eslint.
Else we could have the same content but  different styles by different people, which is what a linter resolves.

Reply:
Thanks for the explanation. Our lint npm script only looks at ts file in src, so files like .eslintrc.js and gulpfile.js are not linted. But the eslint extension doesn't know this and shows this error 
<img width="661" alt="image" src="https://user-images.githubusercontent.com/323878/192341083-a98759e5-286b-4084-856e-dd81c8521a12.png">
 because they are not included in a tsconfig file. If we want to lint these files it seems we need a tsconfig for each one, or a different eslintrc for them that isn't ts-based, I think. We could continue to ignore the errors for now and clean it up later

Reply:
Got it, lets just ignore this for now.